Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/internal/AppConfigSource.kt

```diff
@@ -24,6 +24,8 @@ class AppConfigSource @Inject constructor(
         if (localConfig != null && localConfig.isValid(timeStamper.nowUTC)) {
             Timber.tag(TAG).d("Returning local config, still valid.")
             return localConfig
+        } else {
+            Timber.tag(TAG).d("Local app config was unavailable(${localConfig == null} or invalid.")
```

Comment:
i would prefer severity 'warn' not 'debug'

Reply:
Maybe invalid is too strong of a word here, the local config is invalid if it older than the CDNs cache header specifies, which is currently just 5min, so we would get the warning a lot.